Week3/homework/step2-3.js

```diff
@@ -5,21 +5,27 @@ function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  for (let i = 0; i < num; i++) {
+    result = +str;
```

Comment:
Here is the bug. The correct shorthand should be:

`result += str`
or 
`result = result + str`